open-company/open-company-web
# Table of Contents

card: https://trello.com/c/WfDyVhCL

To test:
- use `categories` branch on API (i'll finish that review later today)
- check that the ToC lists all the categories and sections
- click on a section and check that it scroll animatedly to the right section
- run the tests or check the CI output
- merge
